src/Features/CSharp/Portable/ConvertToRawString/ConvertRegularStringToRawStringCodeRefactoringProvider.cs

```diff
@@ -37,12 +41,25 @@ private enum ConvertToRawKind
             MultiLineWithoutLeadingWhitespace,
         }
 
+        private static readonly BidirectionalMap<ConvertToRawKind, string> s_kindToEquivalenceKeyMap =
+            new(new[]
+            {
+                KeyValuePairUtil.Create(ConvertToRawKind.SingleLine,
+                                        nameof(CSharpFeaturesResources.Convert_to_raw_string) + "-" + ConvertToRawKind.SingleLine),
+                KeyValuePairUtil.Create(ConvertToRawKind.MultiLineIndented,
+                                        nameof(CSharpFeaturesResources.Convert_to_raw_string)),
+                KeyValuePairUtil.Create(ConvertToRawKind.MultiLineWithoutLeadingWhitespace,
+                                        nameof(CSharpFeaturesResources.without_leading_whitespace_may_change_semantics)),
+            });
+
         [ImportingConstructor]
-        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        [SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification = "Used in test code: https://github.com/dotnet/roslyn/issues/42814")]
```

Comment:
This was unfortunately needed because FixAll testing support for code refactorings is only present in the legacy test framework as of now. Once we address https://github.com/dotnet/roslyn/issues/60703 to expose public APIs for FixAll for refactorings, we should update `Microsoft.CodeAnalysis.Testing` library to add test support for it. Tagging @sharwell as FYI